accumulo/README.md

```diff
@@ -36,7 +36,38 @@ Git clone YCSB and compile:
     cd YCSB
     mvn -pl com.yahoo.ycsb:aerospike-binding -am clean package
 
-### 3. Load Data and Run Tests
+### 3. Create the Accumulo table
```

Comment:
Looks like >=1.7 would work with what's in the readme, with table durability and the server-side mutation buffer lacking prior. I've tried to clarify this.